packages/ruleset/src/functions/api-symmetry.js

```diff
@@ -145,60 +145,43 @@ function checkForGraphFragmentPattern(
   schemaFinder
 ) {
   let variantOmitsProperties = false;
-  let isValidGraphFragment = true;
```

Comment:
This didn't necessarily cause problems but it did make debugging a lot harder, since once this got set, every call to `isGraphFragment` would return false. Now, we use a local variable which is probably a more clean and appropriate overall.